src/main/java/uk/gov/hmcts/darts/authentication/controller/AuthenticationCommonController.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.darts.authentication.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import org.springframework.web.bind.annotation.GetMapping;
+import uk.gov.hmcts.darts.authorisation.model.UserState;
+
+public interface AuthenticationCommonController {
+
+    @Operation(
+        tags = {"Authentication"}
+    )
```

Comment:
Would be worth adding `description` and/or `summary` values here to enhance the generated swagger docs.

Reply:
added